pkg/compliance/spec/compliance.go

```diff
@@ -70,18 +73,32 @@ func scannerByCheckID(checkID string) types.Scanner {
 	}
 }
 
-// GetComplianceSpec accepct compliance flag name/path and return builtin or file system loaded spec
-func GetComplianceSpec(specNameOrPath string) (ComplianceSpec, error) {
+// GetComplianceSpec accept compliance flag name/path and return builtin or file system loaded spec
+func GetComplianceSpec(specNameOrPath string, c cache.TrivyCache) (ComplianceSpec, error) {
```

Comment:
I had passed in an interface based on [this](https://bryanftan.medium.com/accept-interfaces-return-structs-in-go-d4cab29a301b) principle of Go. WDYT?

Reply:
I think the interface is redundant here, as it is only introduced for tests. Since all paths depend only on the cache folder, they can be resolved locally.

Reply:
Understood, update it here: https://github.com/aquasecurity/trivy/pull/7179/commits/fe98ea6c5b172af9701979439cbda173680127bb